helix-core/src/main/java/org/apache/helix/model/ClusterConfig.java

```diff
@@ -134,7 +136,13 @@
     // offline for more than this specified time period, and users call purge participant API,
     // then the node will be removed.
     // The unit is milliseconds.
-    OFFLINE_DURATION_FOR_PURGE_MS
+    OFFLINE_DURATION_FOR_PURGE_MS,
+
+    // The following 3 keywords are for metadata in batch disabled instance
+    HELIX_ENABLED_TIMESTAMP,
```

Comment:
I'm thinking if this will be confusing for some users. `HELIX_ENABLED_DISABLED_TIMESTAMP` might be better?

Reply:
Good point, updated.